srai/embedders/highway2vec/model.py

```diff
@@ -0,0 +1,85 @@
+"""
+Embedding model for Highway2Vec.
+
+This module contains the embedding model from the `highway2vec` paper [1].
+
+References:
+    [1] https://doi.org/10.1145/3557918.3565865
+"""
+import pytorch_lightning as pl
+import torch
+import torch.nn.functional as F
+from torch import nn, optim
+
+
+class Highway2VecModel(pl.LightningModule):  # type: ignore
+    """Autoencoder based embedding model for highway2vec."""
+
+    def __init__(self, in_dim: int, hidden_dim: int = 64, latent_dim: int = 30, lr: float = 1e-3):
```

Comment:
How about keeping those params named the same as in gtfs2vec?
```suggestion
    def __init__(self, n_features: int, n_hidden: int = 64, n_embed: int = 30, lr: float = 1e-3):
```